### DISCUSSION TITLE
音频播放换个播放器如何？

### DISCUSSION THREAD

**Discussion by alhricke** [spencerwooo/onedrive-vercel-index#704] [Ideas]
![image](https://user-images.githubusercontent.com/20869340/170615720-0c5afc4c-c7a0-4488-86ee-7572b950bfb7.png)
之前在GitHub上看到这个[PanIndex](https://github.com/libsgh/PanIndex)项目上的音乐播放器，可以列出当前文件夹里面所有的可播放音频文件，可以自动的顺序播放文件夹里面的音频……

**Comment by alhricke** [comment]
![1653618587(1)](https://user-images.githubusercontent.com/20869340/170616862-f73f2f9c-03f0-434b-bb94-19116f9eb46c.png)
还有里面的视频播放器也是有列表的，看起来功能较为丰富……

**Comment by Seele0oO** [comment]
目前也遇到了播放一些视频文件只有声音没有画面的情况……

**Comment by alhricke** [comment]
[alist](https://github.com/alist-org/alist)；这个项目里面的音频播放器和视频播放器也挺不错的……

**Comment by H2Sxxa** [comment]
赞同，自动播放和播放列表感觉还是很方便的。。。

**Comment by H2Sxxa** [comment]
赞同，自动播放和播放列表感觉还是很方便的。。。
